Dockerfile

```diff
@@ -12,7 +12,10 @@ WORKDIR /app
 COPY package*.json ./
 
 # Install dependencies
-RUN npm install
+# RUN npm install
+RUN yarn set version berry
```

Comment:
Please change `RUN yarn set version berry` to `RUN yarn set version 4.1.0` that way during the Docker build it won't download the latest version of Yarn and cause a mismatch error because the build has the latest version but the `package.json` has V 4.1.0.

Reply:
Done.